Source/WebKit/UIProcess/API/C/WKWebsiteDataStoreRef.cpp

```diff
@@ -949,3 +875,78 @@ void WKWebsiteDataStoreGetAllStorageAccessEntries(WKWebsiteDataStoreRef dataStor
     callback(context, WKMutableArrayCreate());
 #endif
 }
+
+WKWebsiteDataTypes WKWebsiteDataStoreAllWebsiteDataTypes()
+{
+    return kWKWebsiteDataTypesCookies
+        | kWKWebsiteDataTypesDiskCache
+        | kWKWebsiteDataTypesMemoryCache
+        | kWKWebsiteDataTypesOfflineWebApplicationCache
+        | kWKWebsiteDataTypesSessionStorage
+        | kWKWebsiteDataTypesLocalStorage
+        | kWKWebsiteDataTypesWebSQLDatabases
+        | kWKWebsiteDataTypesIndexedDBDatabases
+        | kWKWebsiteDataTypesMediaKeys
+        | kWKWebsiteDataTypesSearchFieldRecentSearches
+#if ENABLE(SERVICE_WORKER)
+        | kWKWebsiteDataTypesServiceWorkerRegistrations
+#endif
+        | kWKWebsiteDataTypesDOMCache
+        | kWKWebsiteDataTypesDeviceIdHashSalt
+        | kWKWebsiteDataTypesFileSystem;
+}
+
+WKWebsiteDataTypes WKWebsiteDataStoreAllWebsiteDataTypesIncludingPrivate()
```

Comment:
Let's just include all of them in WKWebsiteDataStoreAllWebsiteDataTypes and not have this function.

Reply:
OK. I'll fix it.